### PR TITLE
refactor: Drop redundant columns from `teams` table

### PR DESCRIPTION
**Pull request by RODO94**
# What does this PR do?

I have dropped all columns in the `teams` table that have been moved to the `team_settings` table. I have updated the necessary scripts in seed-database/write.

Ran GH regression tests and all passed

### CODE REVIEW

**Comment by RODO94** [comment]
[Allow editors to see/update their team's submission email in "Team settings" form](https://trello.com/c/dVsTQpZZ/2984-allow-editors-to-see-update-their-teams-submission-email-in-team-settings-form)

**Review by jessicamcinchak — COMMENTED** [review]
Two adjustments please! 
- Can you please squash migrations into a single `up.sql` & `down.sql` ? 
  - You can drop many columns at once on the same table like `ALTER TABLE table DROP COLUMN col1, DROP COLUMN col2;` 
- Similar to comment on #3635, the order of commands in the `down` files are not currently valid - eg need to add a column before it can be commented on, altered etc

**Review by DafyddLlyr — CHANGES_REQUESTED** [review]
Please also take a look at the migration files at `planx-new/hasura.planx.uk/migrations/1692010483148_add_team_boundary` - there's an orphaned `boundary_bbox()` function that should also be tidied up here now we're handling this on the frontend.

One thing to consider here - just updating db columns won't update the types directly. We'll also need a corresponding change in `planx-core` which removes these properties from the `Team` type (if they're present), and then import this type into the wider codebase to pick up any cascading changes. There is a chance that all tests are passing because types and mocks are unaware of the changes made in this PR.

File: hasura.planx.uk/metadata/tables.yaml [hunk 1]
```diff
@@ -2127,38 +2106,9 @@
...
           - domain
           - name
-          - notify_personalisation
-          - settings
           - slug
-          - submission_email
         filter: {}
         check: null
-- table:
-    name: teams_summary
```
[the remark this answers is not in the document]
Comment (RODO94) [reply hasura.planx.uk/metadata/tables.yaml:2137]: Sorry! That was in the submission email PR, but this is the exact same issue. Removing columns and not repointing the View creation. My bad for missing this again

[the remark this answers is not in the document]
Comment (RODO94) [reply hasura.planx.uk/metadata/tables.yaml:2137]: @DafyddLlyr for this one, if we merge my submission_email PR (#3640), I have already changed the references in the View creation, so this won't be an issue? 

Could be a nice candidate to re-do this on a clean branch once #3640 is merged and reopen a new PR?

[the remark this answers is not in the document]
Comment (RODO94) [reply hasura.planx.uk/metadata/tables.yaml:2137]: Otherwise, I would be repeating code across the migrations

**Pull request closed without merging**